apps/user-profile/src/components/modules/UserOnboardingModal/StyledUserOnboardingModal.js

```diff
@@ -8,11 +9,35 @@ export const ModalContainer = styled.div`
 
 export const ModalHeader = styled.div`
   width: 100%;
+  display: flex;
+  align-items: center;
+  background-color: ${theme.colors.Black};
+  color: #F0EDEE;
+  font-weight: 400;
+
+  img {
+    margin-right: 1rem;
+    margin-left: 1rem;
+  }
+  /* GREYSCALE_OFF_WHITE */
   ${'' /* Enter styling */}
 `;
 
 export const ModalBody = styled.div`
-  padding: 0 120px;
+  padding: 0 115px;
+
+h2 {
```

Comment:
Gonna get you to make some changes here to reflect the changes I'm planning for next week, I'll speak to you about it.